core/src/main/java/org/akaza/openclinica/service/crfdata/BeanPropertyService.java

```diff
@@ -178,8 +178,7 @@ private void executeAction(Object result,PropertyBean propertyBean,ExpressionBea
 				studyEvent.setDateStart(df.parse((String) result));
 
 			} catch (ParseException e) {
-				e.printStackTrace();
-				LOGGER.info(e.getMessage());
+				LOGGER.info("Date is not getting parsed properly: ",e);

```

Comment:
this should be at the error level

Reply:
I have changed according to your comments